action.yml

```
@@ -65,6 +65,10 @@ runs:
           fi
         fi
         helm_ut_plugin_version="${{ inputs.unittest-version }}"
+        # if helm_ut_plugin_version is "latest" set it to empty string
+        if [ "$helm_ut_plugin_version" = "latest" ]; then
+          helm_ut_plugin_version=""
+        fi
         if [ -n "$helm_ut_plugin_version" ]; then
```

The `$helm_ut_plugin_version` variable is only used here in this script, so I make the following style suggestion:

```suggestion
        if [ -n "$helm_ut_plugin_version" ] && [ "$helm_ut_plugin_version" != "latest" ]; then
```

---

Yeah, I was going back and forth on which way to go with this.